src/main/resources/swagger/listings/entities

```diff
@@ -552,8 +552,12 @@
 		"AddUpdateAttribute": {
 			"id": "AddUpdateAttribute",
 			"description": "",
-			"required": ["attributeName", "addUpdateAttribute"],
+			"required": ["op", "attributeName", "addUpdateAttribute"],
 			"properties": {
+				"op": {
+					"type": "string",
+					"description": ""
```

Comment:
No, but these models are likely to change in some way for my next PR where we'll (hopefully) support displaying complex models.